packages/core-database-sequelize/__tests__/repositories/transactions.test.js

```diff
@@ -20,8 +20,12 @@ const getWallet = address => {
   return spv.walletManager.getWalletByAddress(address)
 }
 
-beforeAll(async () => {
+beforeAll(async (done) => {
```

Comment:
This callback could be removed

Reply:
I copied it from another test: https://github.com/ArkEcosystem/core/blob/master/packages/core-database/__tests__/interface.test.js#L8

But I can remove it.